controllers/spec/utils.go

```diff
@@ -246,11 +246,12 @@ func generateSourceOutputSpec(source *v1alpha1.Source) *proto.SinkSpec {
 		}
 	}
 	return &proto.SinkSpec{
-		TypeClassName:  source.Spec.Output.TypeClassName,
-		Topic:          source.Spec.Output.Topic,
-		ProducerSpec:   &producerSpec,
-		SerDeClassName: source.Spec.Output.SinkSerdeClassName,
-		SchemaType:     source.Spec.Output.SinkSchemaType,
+		TypeClassName:                source.Spec.Output.TypeClassName,
+		Topic:                        source.Spec.Output.Topic,
+		ProducerSpec:                 &producerSpec,
+		SerDeClassName:               source.Spec.Output.SinkSerdeClassName,
+		SchemaType:                   source.Spec.Output.SinkSchemaType,
+		ForwardSourceMessageProperty: *source.Spec.ForwardSourceMessageProperty,
```

Comment:
The webhook is not available in tests, so `ForwardSourceMessageProperty` might be nil if not provided.

Reply:
I thought I add a webhook for the check. Let me know if this helps

Reply:
yes, but the webhook is disabled by default, and needs some extra configs to make it work. I was working on that with #238, before the webhook is working, we can consider adding [kubebuilder CRD validation markers](https://book.kubebuilder.io/reference/markers/crd-validation.html).